keyboards/keychron/q1/ansi_stm32l432_encoder/info.json

```diff
@@ -0,0 +1,116 @@
+{
+    "keyboard_name": "Keychron Q1",
+    "manufacturer": "Keychron",
+    "url": "https://github.com/Keychron",
+    "maintainer": "lalalademaxiya1",
+    "usb": {
+        "vid": "0x3434",
+        "pid": "0x0107",
+        "device_version": "2.0.0"
+    },
+    "matrix_pins": {
+        "cols": ["C14", "C15", "A0", "A1", "A2", "A3", "A4", "A5", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN"],
```

Comment:
`null` could (should) be used instead of `"NO_PIN"`.  *(same on the other `json`'s)*

Reply:
All right, Thanks!

Reply:
Hi, I try to compile it with this way but it was still failed, and displayed that "'null' is not valid under any of the given schemas" in terminal.